content/blog/urbit-for-normies.md

```diff
@@ -0,0 +1,181 @@
++++
+title = "Urbit for Normies"
+date = 2020-02-12
+description = "What is Urbit and why should you care? A layperson’s guide to the coming new internet."
```

Comment:
I think cutting this to second sentence is preferable for Twitter cards and the like.

Reply:
K — also realizing i need to fix the preview card too

Reply:
Ah, yes I see it’s the sigil one. Were you adding an image at the top of the article too? Nothing there atm

Reply:
I'm ok with keeping it a textual opening — I don't think "image at top" is a hard rule we need to stick  by for all essay content fwiw, would be down to discuss/argue this point though.

Reply:
Currently structured, I was basically going for two images splitting the essay into three sections more or less